### PR TITLE
Set OCaml to 4.14.2

### PR DESCRIPTION
**Pull request by cuihtlauac**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
           - macos-latest
           - ubuntu-latest
         ocaml-compiler:
-          - ocaml-base-compiler.4.14.1
+          - ocaml-base-compiler.4.14.2
```
Comment (smorimoto) [inline .github/workflows/ci.yml:25]:
```suggestion
          - “4.14”
```

Comment (smorimoto) [inline .github/workflows/ci.yml]:
```suggestion
          - “4.14”
```

File: .github/workflows/debug-ci.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
           - macos-latest
           - ubuntu-latest
         ocaml-compiler:
-          - ocaml-base-compiler.4.14.1
+          - ocaml-base-compiler.4.14.2
```
Comment (smorimoto) [inline .github/workflows/debug-ci.yml]:
```suggestion
          - “4.14”
```

File: .github/workflows/linkcheck.yml [hunk 3]
```diff
@@ -21,7 +21,7 @@ jobs:
         os:
           - ubuntu-latest
         ocaml-compiler:
-          - ocaml-base-compiler.4.14.1
+          - ocaml-base-compiler.4.14.2
```
Comment (smorimoto) [inline .github/workflows/linkcheck.yml]:
```suggestion
          - “4.14”
```

File: .github/workflows/scrape.yml [hunk 4]
```diff
@@ -18,7 +18,7 @@ jobs:
         os:
           - ubuntu-latest
         ocaml-compiler:
-          - ocaml-base-compiler.4.14.1
+          - ocaml-base-compiler.4.14.2
```
Comment (smorimoto) [inline .github/workflows/scrape.yml]:
```suggestion
          - “4.14”
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (cuihtlauac) [reply .github/workflows/ci.yml]: Much nicer, thanks for the suggestion

**Comment by cuihtlauac** [comment]
@smorimoto: The change you proposed fails

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -22,10 +22,10 @@ jobs:
           - macos-latest
           - ubuntu-latest
         ocaml-compiler:
-          - ocaml-base-compiler.4.14.1
+          - “4.14”
```
Comment (panglesd) [inline .github/workflows/ci.yml]: Just passing by, could not resist :)
```suggestion
          - "4.14"
```

File: .github/workflows/debug-ci.yml [hunk 6]
```diff
@@ -16,10 +16,10 @@ jobs:
           - macos-latest
           - ubuntu-latest
         ocaml-compiler:
-          - ocaml-base-compiler.4.14.1
+          - “4.14”
```
Comment (cuihtlauac) [inline .github/workflows/debug-ci.yml]:
```suggestion
          - "4.14"
```

File: .github/workflows/linkcheck.yml [hunk 7]
```diff
@@ -21,7 +21,7 @@ jobs:
         os:
           - ubuntu-latest
         ocaml-compiler:
-          - ocaml-base-compiler.4.14.1
+          - “4.14”
```
Comment (cuihtlauac) [inline .github/workflows/linkcheck.yml]:
```suggestion
          - "4.14"
```

File: .github/workflows/scrape.yml [hunk 8]
```diff
@@ -18,7 +18,7 @@ jobs:
         os:
           - ubuntu-latest
         ocaml-compiler:
-          - ocaml-base-compiler.4.14.1
+          - “4.14”
```
Comment (cuihtlauac) [inline .github/workflows/scrape.yml]:
```suggestion
          - "4.14"
```

**Comment by cuihtlauac** [comment]
Thanks, @panglesd, for pointing out the issue with quotes.

**Comment by smorimoto** [comment]
Suggestions from mobile don't seem to work 😑

**Review by sabine — APPROVED** [review]

**Pull request merged**